test/functional/fixtures/regression/gh-2153/pages/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <script type="text/javascript">
+        window.counter = 0;
+
+        function onClick (e) {
+            var selectEl = document.getElementById('select');
+            var optOne = document.getElementById('one');
```

Comment:
Test contains click only on the `option` with text `"Two"`.
Let's do not check the `one` option here, e.g.
```js
if (select !== e.target && optTwo !== e.target)
    throw new Error('Shadow element appeared in user event handler');
```